ar.com.pilas_engine.App.yaml

```diff
@@ -17,20 +17,13 @@ modules:
   - name: extras
     buildsystem: simple
     build-commands:
-      - 'install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml'
-      - 'install -Dm644 ${FLATPAK_ID}.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg'
-      - 'install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop'
+      - 'install -Dm644 extras/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml'
+      - 'install -Dm644 extras/${FLATPAK_ID}.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg'
+      - 'install -Dm644 extras/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop'
     sources:
-      - type: file
```

Comment:
you should remove the files from the repository as well.

Reply:
Ah good to know. How can I do that? Using `cleanup` in the manifest?

Reply:
`git rm` in the repository and commit

Reply:
ah those files. Of course, thanks.